apis/r/tests/testthat/test-shape.r

```diff
@@ -31,9 +31,9 @@ test_that("SOMADataFrame shape", {
     sdf <- SOMADataFrameOpen(uri)
 
     if (.new_shape_feature_flag_is_enabled()) {
-      expect_true(sdf$has_upgraded_domain())
+      expect_true(sdf$tiledbsoma_has_upgraded_domain())
```

Comment:
Hate to nitpick, but can we be consistent with file endings? This is the one file in the package using `.r` instead of `.R`

Reply:
> Hate to nitpick

Not at all, I appreciate your eyes! :) 

> Hate to nitpick, but can we be consistent with file endings? This is the one file in the package using .r instead of .R

I simply didn't notice -- thanks for pointing it out!